opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java

```diff
@@ -645,6 +646,19 @@ private void updateApStatus(Equipment ce, ConnectNodeInfo connectNodeInfo) {
             networkAdminStatusRec.setDetails(netAdminStatusData);
 
             networkAdminStatusRec = statusServiceInterface.update(networkAdminStatusRec);
+            
+            Status channelStatusRec = statusServiceInterface.getOrNull(ce.getCustomerId(), ce.getId(),
+                    StatusDataType.RADIO_CHANNEL);
+            if (channelStatusRec == null) {
+            	channelStatusRec = new Status();
+            	channelStatusRec.setCustomerId(ce.getCustomerId());
+            	channelStatusRec.setEquipmentId(ce.getId());
+            	channelStatusRec.setStatusDataType(StatusDataType.RADIO_CHANNEL);
+                EquipmentChannelStatusData channelStatusData = new EquipmentChannelStatusData();
```

Comment:
This creates an empty status, with no channel info. 
The channels will only be populated after wifiRadioStatusDbTableUpdate - when channels are changed. 
Does it make sense to read the OpensyncAPRadioState in here and populate the channels, so there's no empty values initially?

Reply:
I will create it when it is ready. Thanks.

Reply:
If it's not ready now - please create a JIRA to be addressed in a future

Reply:
Just made the change, now RADIO_CHANNEL status will be created only when there is data.